arkane/statmech.py

```diff
@@ -234,7 +234,7 @@ def load(self, pdep=False, plot=False):
         path = self.path
         directory = os.path.abspath(os.path.dirname(path))
 
-        def create_log(log_path):
+        def create_log(log_path, check_for_errors=True):
             if not os.path.isfile(log_path):
```

Comment:
Thanks for adding this! Can you also modify one of the Arkane example files so that this gets tested out when we run the  functional tests? Add a comment in the input file letting the user know:

1. That this is the syntax for instructing Arkane to ignore any error messages in the file
2. This is not actually needed in this specific file, but added to show the syntax

Reply:
Done! I modified `testing/arkane/two_parameter_arrhenius_fit/CH3OO.py`